flow/core/rewards.py

```diff
@@ -322,6 +322,8 @@ def energy_consumption(env, gain=.001):
     rho = 1.225  # air density (kg/m^3)
     A = 2.6  # vehicle cross sectional area (m^2)
     for veh_id in env.k.vehicle.get_ids():
+        if veh_id not in env.k.vehicle.previous_speeds:
+            continue
```

Comment:
This accounts fo the fact the power used at the first step is really high (because the previous speed is assumed to be 0. It's a bit of a hack, but it should be fixed by altering the energy model to not do finite differencing.